Automattic/wp-calypso
# Akismet Pro checkout line item prices should reflect license quantity

This PR Adjusts how prices are displayed in checkout for the Akismet Pro 500 product which has a _licensed quantity_, in order to make the price/total a little more clear to users.  (See before & after images:)

BEFORE | AFTER
--- | ---
![Markup 2023-12-02 at 09 18 03](https://github.com/Automattic/wp-calypso/assets/11078128/4184d7da-701c-40f7-912a-008ea8dabd46) | ![Markup 2023-12-02 at 09 15 58](https://github.com/Automattic/wp-calypso/assets/11078128/7f0ba9a6-1e1a-428f-89e9-49b6e93efa39)




- Related discussion concerning dropdown price display: pbNhbs-8P6-p2#comment-18986
- PT: Akismet Checkout – Multiple licenses: pbNhbs-8My-p2
- Akismet Checkout: Multiple Licenses – Implementation Plan: pbNhbs-8P6-p2

## Testing Instructions

- (if not already merged), first apply patch D128326-code to your sandbox and sandbox `public-api.wordpress.com`.
- Checkout this PR branch and `yarn-start`
- Go to https://akismet.com/pricing and select/click the  "Get Pro" button. 
- On the checkout page, in the URL, replace `https://wordpress.com` with `http://calypso.localhost:3000` and press enter.
- You should see the "Number of licenses" dropdown, like in the image above.
- Select a different number of sites/licenses.
- Verify the way the prices are displayed make a little more sense.

**Note:** There is probably room for improvement here, and we probably need Design to iterate on the optimum price display, however for now I think we're just shooting for "acceptable" in order to get this project shipped and in front of our users asap (Targeting Monday Dec 4th).

## Pre-merge Checklist
